mappings/ctv3sctmap2/mappers.py

```diff
@@ -1,7 +1,8 @@
 from collections import defaultdict
 
+from coding_systems.ctv3 import coding_system as ctv3
 from coding_systems.ctv3.models import Concept as CTV3Concept
-from coding_systems.snomedct.coding_system import descendant_relationships, lookup_names
+from coding_systems.snomedct import coding_system as snomed
```

Comment:
For consistency, can we import this as `snomedct`?

Reply:
Shoot, sorry, yes - thought I'd fixed all of these!

Reply:
Fixed in 8df5628